### ISSUE TITLE
ParseError: Found both non-rdf:* property attributes and rdf:datatype

### ISSUE THREAD

**Issue by marcelomachado** [rdfjs/rdfxml-streaming-parser.js#21]
I am trying to parse the following RDF/XML file:
```
<?xml version="1.0"?>
<RDF xmlns="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
xmlns:owl="http://www.w3.org/2002/07/owl#"
xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
xmlns:xs="http://www.w3.org/2001/XMLSchema#"
xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
>
    <Description rdf:about="http://x.p.t/o/TBox#hasAnswerText">
        <rdfs:range rdf:resource="http://www.w3.org/2001/XMLSchema#string"/>
        <rdfs:domain rdf:resource="http://x.p.t/o/TBox#Answer"/>
        <rdf:type rdf:resource="http://www.w3.org/2002/07/owl#DatatypeProperty"/>
    </Description>

    <Description rdf:about="http://x.p.t/o/ABox#xpto">
        <ns18:hasAnswerText  xmlns:ns18="http://x.p.t/o/TBox#" rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Yes</ns18:hasAnswerText>
        <rdf:type rdf:resource="http://www.w3.org/2002/07/owl#NamedIndividual"/>
        <rdf:type rdf:resource="http://x.p.t/o/TBox#Answer"/>
    </Description>
</RDF>
```
However the following error occurs:

```
ParseError: Found both non-rdf:* property attributes and rdf:datatype (http://www.w3.org/2001/XMLSchema#string).
```

I moved the `xmlns:ns18="http://x.p.t/o/TBox#"` attribute to `RDF` tag and everything works fine, but I have no control to change this XML.

**Comment by rubensworks** [comment]
I think this document is invalid RDF/XML, but other parsers seem to accept this, so I may be misinterpreting something in the spec.
I'll have to look into it.

**Comment by rubensworks** [comment]
Fixed in 1.3.2.
